src/app_charts/base/cloud/cert-manager-issuers.yaml

```diff
@@ -16,7 +16,7 @@ spec:
 {{ if eq .Values.certificate_provider "lets-encrypt" }}
 ---
```

Comment:
Please add a comment such as:
# We need kind to be ClusterIssuer to be able to issue verts for multiple namespaces. If all we need would be the default namespace Issuer would be sufficient.